### PR TITLE
feat: add support for localization

### PR DESCRIPTION
**Pull request by swimmingkiim**
## Things to check before merge
- `initializeDateFormatting` had been removed in this PR. I found that when saving data to firebase, it uses related methods. Please check this code is fine with when it comes to saving data to firebase firestore.
- README should be updated. After updating or adding any `.arb` files, developer should run `flutter gen-l10n` inside of root project directory.

## References
- [Removing `initializeDateFormatting` since it conflicts with `flutter_localizations`](https://stackoverflow.com/a/68911879)
- [Official Flutter Document for `Internationalizing Flutter apps`](https://docs.flutter.dev/accessibility-and-localization/internationalization)
- [Official Flutter Document for adding custom localization messages](https://docs.flutter.dev/accessibility-and-localization/internationalization#adding-your-own-localized-messages)

### CODE REVIEW

File: l10n.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+arb-dir: lib/l10n
+template-arb-file: app_ja.arb
+output-localization-file: app_localizations.dart
```
Comment (kur00) [inline l10n.yaml:3]: [WANT]
need a new line

File: lib/l10n/app_ja.arb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+  "home": "ホーム",
+  "calendar": "カレンダー",
+  "print": "印刷",
+  "help": "ヘルプ"
+}
```
Comment (kur00) [inline lib/l10n/app_ja.arb:6]: WANT
New line too

File: lib/l10n/app_ko.arb [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
+  "home": "홈",
+  "calendar": "달력",
+  "print": "출력",
+  "help": "도움말"
+}
```
Comment (kur00) [inline lib/l10n/app_ko.arb:6]: WANT
New line too

**Review by kur00 — COMMENTED** [review]
Thx for your quick coding!!
LGTM for logic of it but I want you to fix some small issue.

**Review by ayuki-joto — APPROVED** [review]

**Pull request merged**